packages/client/src/types/unit.ts

```diff
@@ -29,6 +38,7 @@ type TUnit = {
   label?: string;
   type?: string; // should be an enum
   updated_at?: string;
-} & Record<string, unknown>;
+} & Record<string, unknown> &
+  UnitLinkDetail<WithLink>;
```

Comment:
This is useful for describing units that has link info within data. Every time that a unit is retrieved from topology those will be filled. 

In order to use it within type declaration just write:

`TUnit<true>`